Controller/Dashboard.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\Controller;
+
+use KaroIO\MessengerMonitorBundle\FailedMessage\FailedMessageRepository;
+use KaroIO\MessengerMonitorBundle\Locator\ReceiverLocator;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Messenger\Transport\Receiver\MessageCountAwareInterface;
+use Twig\Environment;
+
+class Dashboard
+{
+    private $twig;
+    private $receiverLocator;
+    private $failedMessageRepository;
+
+    public function __construct(
+        Environment $twig,
+        ReceiverLocator $receiverLocator,
+        FailedMessageRepository $failedMessageRepository
+    ) {
+        $this->twig = $twig;
+        $this->receiverLocator = $receiverLocator;
+        $this->failedMessageRepository = $failedMessageRepository;
+    }
+
+    public function __invoke(): Response
+    {
+        $transports = [];
+        foreach ($this->receiverLocator->getReceiversMapping() as $name => $receiver) {
+            $transports[$name] = $receiver instanceof MessageCountAwareInterface ? $receiver->getMessageCount() : -1;
```

Comment:
`null` might be a better choice than `-1`?

Reply:
sure, i've used the same value than the command `messenger:monitor` without thinking much more thant copy/pasting